{
    "url": "https://api.github.com/repos/pallets/flask/issues/600",
    "repository_url": "https://api.github.com/repos/pallets/flask",
    "labels_url": "https://api.github.com/repos/pallets/flask/issues/600/labels{/name}",
    "comments_url": "https://api.github.com/repos/pallets/flask/issues/600/comments",
    "events_url": "https://api.github.com/repos/pallets/flask/issues/600/events",
    "html_url": "https://github.com/pallets/flask/issues/600",
    "id": 7152742,
    "node_id": "MDU6SXNzdWU3MTUyNzQy",
    "number": 600,
    "title": "cannot run testsuite on Windows",
    "user": {
        "login": "petef4",
        "id": 2046639,
        "node_id": "MDQ6VXNlcjIwNDY2Mzk=",
        "avatar_url": "https://avatars0.githubusercontent.com/u/2046639?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/petef4",
        "html_url": "https://github.com/petef4",
        "followers_url": "https://api.github.com/users/petef4/followers",
        "following_url": "https://api.github.com/users/petef4/following{/other_user}",
        "gists_url": "https://api.github.com/users/petef4/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/petef4/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/petef4/subscriptions",
        "organizations_url": "https://api.github.com/users/petef4/orgs",
        "repos_url": "https://api.github.com/users/petef4/repos",
        "events_url": "https://api.github.com/users/petef4/events{/privacy}",
        "received_events_url": "https://api.github.com/users/petef4/received_events",
        "type": "User",
        "site_admin": false
    },
    "labels": [],
    "state": "closed",
    "locked": false,
    "assignee": null,
    "assignees": [],
    "milestone": null,
    "comments": 0,
    "created_at": "2012-09-26T15:16:16Z",
    "updated_at": "2012-10-07T14:04:33Z",
    "closed_at": "2012-10-07T14:04:33Z",
    "author_association": "NONE",
    "body": "Attempting to run tests of Flask itself fails immediately.\n\n```\n(python2.7) C:\\Users\\peter\\repos\\mitsuhiko\\flask.git>python run-tests.py\nError: 'module' object has no attribute 'samefile'\n```\n\nThis is due to a call to os.path.samefile (in flask/testsuite/__init__.py) which is only available on Unix.\n"
}